src/aave-v3/SupplyVault.sol

```diff
@@ -245,7 +246,7 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
         uint256 _userBalance,
         uint128 _indexAccrual
     ) internal pure returns (uint128 unaccruedReward) {
-        unaccruedReward = _userBalance.mulDivDown(_indexAccrual, SCALE).safeCastTo128();
+        unaccruedReward = _userBalance.rayMul(_indexAccrual).safeCastTo128();
```

Comment:
Rounds up instead of rounding down. I think that it was intended.

Reply:
This was due to issues with round up. We don't have the round-down functionality on the `WadRayMath` lib... but here it does seem to bring any issue

Reply:
So shall we use mulDivDown with a RAY then?

Reply:
IMO we don't really need to

Reply:
@MathisGD ?

Reply:
I think that it should be rounded down, because otherwise you distribute more rewards than what you have. Maybe it is never going to be an issue (considering the amounts that it represent), but by rounding down we are sure that it will be ok.

Reply:
Aave is using `WadRayMath` for information*

Reply:
But is it the same rewards management ?

Reply:
Not exactly of course, but I don't think it brings issue here

Reply:
IMO rounding down is the "correct" solution for a sure reliability of the contract, but realistically I don't foresee this ever being a problem. But for peace of mind, mulDivDown does seem to be the better solution, so I will add this and we can revert if there is disagreement.

Reply:
Ok then let's use `mulDivDown`